tests/tasks/cleanup_candlepin_container.yml

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Check if candlepin container is running
+  command: podman ps -a --filter name=candlepin
+  register: __rhc_candlepin_container
+  changed_when: false
+
+- name: Stop Candlepin container
+  command: podman stop candlepin
+  changed_when: true
+  when: '"candlepin" in __rhc_candlepin_container.stdout'
+
+- name: Remove Candlepin container
+  command: podman rm candlepin
```

Comment:
This gives an error because the container was run with the `--rm` argument, which automatically removes the container after `podman stop`
https://github.com/linux-system-roles/rhc/blob/main/tests/tasks/setup_candlepin.yml#L56

I still think it is a good idea to do the `rm` just to ensure that it is gone - maybe do `podman rm --force --ignore candlepin`

Reply:
Well, if I'm running podman with `--rm`, then I expect the container to be gone on `stop`, and not require and removal manually... If it does not do that, I'd consider it a podman bug.